python/pyproject.toml

```diff
@@ -53,8 +52,7 @@ classifiers = [
 [project.optional-dependencies]
 test = [
     "pytest",
-    "pytest-xdist",
```

Comment:
Looks like this got dropped. Added a suggestion above about this

Reply:
I dropped this intentionally to clean up a bit. rmm can't actually run tests in parallel because many of them require most of the GPU, so we never actually leverage pytest-xdist.